### PR TITLE
36 login flow

### PR DESCRIPTION
**Pull request by aaronpflower**
I cleaned up the login flow to what we discussed the other night.  Then started working/fiddling on other things not part of this issue on this branch, so I figured it would be best to review and merge this, and create some new issues for other UI things.

### CODE REVIEW

File: client/css/global.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.test {
+    color: red;
+};
```
Comment (burtonjc) [inline client/css/global.css]: delete this

File: client/partials/mainContent.html [hunk 2]
```diff
@@ -1,65 +1,36 @@
-<div class="container marketing">
```
Comment (burtonjc) [inline client/partials/mainContent.html]: rename this file to be hyphenated (aka. main-content.html)

File: server/views/index.jade [hunk 3]
```diff
@@ -7,6 +7,8 @@ html(lang="en" ng-app="application")
     script(src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js")
     link(rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css")
     link(rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css")
+    link(rel="stylesheet" type="text/css" href="../../client/css/global.css")
```
Comment (burtonjc) [inline server/views/index.jade]: This path is wrong, we talked about it monday night.

File: server/views/landing.jade [hunk 4]
```diff
@@ -0,0 +1,28 @@
+doctype html
+
+html(lang="en" ng-app="application")
+  head
+    title FoosNet
+    script(src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js")
+    script(src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js")
+    link(rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css")
+    link(rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css")
+    link(rel="stylesheet" type="text/css" href="../../client/css/global.css")
```
Comment (burtonjc) [inline server/views/landing.jade]: path is wrong

**Comment by burtonjc** [comment]
several 404 errors from bad paths on css and images

**Comment by aaronpflower** [comment]
@burtonjc Fixed 404 errors.  Should be ready to merge but let me know if there is anything else that needs to be done first.

**Comment by burtonjc** [comment]
:+1:

**Pull request merged**